components/item-detail/item-detail.tsx

```
@@ -30,43 +28,217 @@ export function ItemDetail({
+                  className="inline-flex items-center px-5 py-2.5 bg-indigo-600 hover:bg-indigo-700 dark:bg-indigo-700 dark:hover:bg-indigo-800 text-white rounded-lg font-medium transition-all duration-200  -md hover: -lg transform hover:-translate-y-0.5 dark: -indigo-900/30"
+                >
+                  <span className="mr-2">🌐</span> Visit Website
+                </a>
+                <button className="inline-flex items-center px-4 py-2.5 bg-gray-100 hover:bg-gray-200 dark:bg-gray-900 dark:hover:bg-gray-800 text-gray-700 dark:text-gray-200 rounded-lg font-medium transition-all duration-200 border border-gray-300 dark:border-gray-800 hover:border-indigo-500/50 dark:hover:border-indigo-700/50">
+                  <svg
+                    className="w-5 h-5 mr-2"
+                    fill="currentColor"
+                    viewBox="0 0 20 20"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path d="M5 4a2 2 0 012-2h6a2 2 0 012 2v14l-5-2.5L5 18V4z"></path>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)